src/server/game/AuctionHouseBot/AuctionHouseBotBuyer.cpp

```diff
@@ -135,11 +139,10 @@ uint32 AuctionBotBuyer::GetItemInformation(BuyerConfiguration& config)
                 itemInfo.MinBuyPrice = std::min(itemInfo.MinBuyPrice, itemBuyPrice);
         }
 
-        // Add/update to EligibleItems if:
-        // has a bid by player or
-        // has no bids and not owned by bot
-        if ((entry->bid && entry->bidder) || (entry->owner && !entry->bid))
-        {
```

Comment:
> where did the bracket go?

That doesn't even match my source file

```
if (!entry->bid || entry->bidder)
```

But you're right, I did accidentally remove a bracket. I was editing directly in github :(